### DISCUSSION TITLE
(Error) Build from Source

### DISCUSSION THREAD

**Discussion by anchitDave** [magda-io/magda#3416] [Q&A]
Hello Team, 
I am trying to build all the micro-services from source but I face this challenge wherein the magda-indexer is not able to install all its dependencies.

Running the build command:
`yarn lerna run build --stream --concurrency=1 --include-dependencies`

Error:
`download error: Caught java.io.IOException: Server returned HTTP response code: 502 for URL: https://dl.bintray.com/monsanto/maven/com/monsanto/labs/mwundo-core_2.12/0.5.0/mwundo-core_2.12-0.5.0.pom (Server returned HTTP response code: 502 for URL: https://dl.bintray.com/monsanto/maven/com/monsanto/labs/mwundo-core_2.12/0.5.0/mwundo-core_2.12-0.5.0.pom) while downloading https://dl.bintray.com/monsanto/maven/com/monsanto/labs/mwundo-core_2.12/0.5.0/mwundo-core_2.12-0.5.0.pom`

Kindly help me with this issue.
Thank you in advance!

**Comment by t83714** [comment]
Please refer to our gitlab pipeline to find out how it's built in our CI:

https://github.com/magda-io/magda/blob/master/.gitlab-ci.yml

Mainly around here: https://github.com/magda-io/magda/blob/0b903349149a60fb484473a2bbf5f22c486906a1/.gitlab-ci.yml#L79
